windows/LiferayNativityShellExtensions/LiferayNativityWindowsUtil/ConfigConstants.h

```diff
@@ -12,6 +12,6 @@
  *  details.
  */
 
-#define REGISTRY_LIFERAY_KEY	L"SOFTWARE\\Liferay Inc\\Liferay Nativity"
+#define REGISTRY_LIFERAY_KEY	L"SOFTWARE\\Box Inc\\Box Desktop"
```

Comment:
There's also still a bunch of variable names that still have Liferay or Nativity in the name